BudgetBudget/View/MonthSelectorView.swift

```diff
@@ -17,7 +17,7 @@ struct MonthSelectorView: View {
     static let currentMonth = months[Calendar.current.dateComponents([.month], from: Date()).month!]
```

Comment:
I just noticed that I have an off by one in here. If we continue using the current month the array access needs a -1

Reply:
Actually noticed that but forgot to fix.